openldap/README.md

```diff
@@ -0,0 +1,349 @@
+        -e LDAP_TLS_CA_FILE=/opt/symas/certs/openldapCA.crt \
+        symas/openldap:latest
+    ```
+
+2. Modifying the `docker-compose.yml` file present in this repository:
+
+    ```yaml
+    services:
+      openldap:
+      ...
+        environment:
+          ...
+          - LDAP_ENABLE_TLS=yes
+          - LDAP_TLS_CERT_FILE=/opt/symas/certs/openldap.crt
+          - LDAP_TLS_KEY_FILE=/opt/symas/certs/openldap.key
+          - LDAP_TLS_CA_FILE=/opt/symas/certs/openldapCA.crt
+        ...
+        volumes:
+          - /path/to/certs:/opt/symas/certs
+          - /path/to/openldap-data-persistence:/openldap/
+      ...
+    ```
+
+### Initializing a new instance
+
+The [Symas OpenLDAP](https://github.com/symas/containers/blob/main/openldap) image allows you to use your custom scripts to initialize a fresh instance.
+
+The allowed script extension is `.sh`, all scripts are executed in alphabetical order and need to reside in `/docker-entrypoint-initdb.d/`.
+
+Scripts are executed are after the initilization and before the startup of the OpenLDAP service.
+
+## Logging
+
+The Symas OpenLDAP Docker image sends the container logs to `stdout`. To view the logs:
+
+```console
+docker logs openldap
+```
+
+You can configure the containers [logging driver](https://docs.docker.com/engine/admin/logging/overview/) using the `--log-driver` option if you wish to consume the container logs differently. In the default configuration docker uses the `json-file` driver.
+
+To see the actual output of the various commands used during setup as well as slapd in the container's logs, set the environment variable `SYMAS_DEBUG=true`. Useful especially to find/debug problems in your configuration that lead to errors causing OpenLDAP not to start.
+
+## Maintenance
+
+### Upgrade this image
+
+Symas provides up-to-date versions of OpenLDAP, including security patches, soon after they are made upstream. We recommend that you follow these steps to upgrade your container.
+
+#### Step 1: Get the updated image
+
+```console
+docker pull symas/openldap:latest
+```
+
+#### Step 2: Stop the running container
+
+Stop the currently running container using the command
+
+```console
+docker stop openldap
+```
+
+#### Step 3: Remove the currently running container
+
+```console
+docker rm -v openldap
+```
+
+#### Step 4: Run the new image
+
+Re-create your container from the new image.
+
+```console
+docker run --name openldap symas/openldap:latest
+```
+
+## Notable Changes
+
+
+## Contributing
+
+We'd love for you to contribute to this container. You can request new features by creating an [issue](https://github.com/symas/containers/issues) or submitting a [pull request](https://github.com/symas/containers/pulls) with your contribution.
+
+## Issues
+
+If you encounter a problem running this container, you can file an [issue](https://github.com/symas/containers/issues/new/choose). For us to provide better support, be sure to fill the issue template.
+
+## License
```

Comment:
Either be more explicit the license notice applies to this file only or mention that software inside the container has own licenses?